tests/test_1d_gradient/results_true.dat

```diff
@@ -1,203 +1,203 @@
 # Iterations: 47
 keff:  1.68873E-01
 fluxes:
-1.970213E+00
-4.070833E-01
-2.320673E+00
-5.841409E-01
-1.937412E+00
-4.023670E-01
+1.970212E+00
```

Comment:
I believe in the past we *sometimes* split at least one track segment. Now we *never* split a segment (as long as the max optical length used to split the segments is based on the max optical path of the existing segments). Thus, not all tests need to be updated.